testing/grepcidr/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Vince Mele <vmele@inoc.com>
+# Maintainer: Vince Mele <vmele@inoc.com>
+pkgname=grepcidr
+pkgver=2.0
+pkgrel=0
+pkgdesc="Filter IPv4 and IPv6 addresses matching CIDR patterns"
+url="http://www.pc-tools.net/unix/grepcidr/"
+arch="all"
+license="GPL3"
+depends=""
+makedepends=""
+install=""
+subpackages=""
+source="http://www.pc-tools.net/files/unix/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	make
+}
+
+check() {
+	cd "$builddir"
+	if [ -x grepcidr ]; then echo OK; else exit 1; fi
```

Comment:
This is not an useful check. Could you run grepcidr, e.g. `./grepcidr --version`?

Reply:
Better yet, it would be good to run grepcidr against some sample data and confirm it behaves as expected.